### PR TITLE
pio-settings: add page

### PR DESCRIPTION
**Pull request by marchersimon**
Part of #5517

### CODE REVIEW

File: pages/common/pio-settings.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pio settings
+
+> View and modify PlatformIO settings.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_settings.html>.
+
+- List all PlatformIO settings:
+
+`pio settings get`
+
+- Show a specific PlatformIO setting:
```
Comment (bl-ue) [inline pages/common/pio-settings.md:10]:
```suggestion
- Display the value of a PlatformIO setting:
```

File: pages/common/pio-settings.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`pio settings get {{setting}}`
+
+- Modify a setting:
```
Comment (bl-ue) [inline pages/common/pio-settings.md:14]:
```suggestion
- Modify the value of a setting (creates the setting if it does not exist):
```

File: pages/common/pio-settings.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`pio settings set {{setting}} {{value}}`
+
+- Reset all settings:
```
Comment (bl-ue) [inline pages/common/pio-settings.md:18]:
```suggestion
- Reset the values of all modifed settings to their factory defaults:
```

File: pages/common/pio-settings.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# pio settings
+
+> View and modify PlatformIO settings.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_settings.html>.
+
+- List all PlatformIO settings:
```
Comment (bl-ue) [inline pages/common/pio-settings.md:6]: I'm not even sure if this is actually what it does, but...
```suggestion
- List the names and values of all PlatformIO settings:
```

File: pages/common/pio-settings.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/pio-settings.md:10]: It doesn't only print the value, but a nice table:
```
Name          Current value [Default]    Description
------------  -------------------------  ------------------------------------
enable_cache  Yes                        Enable caching for HTTP API requests
```

File: pages/common/pio-settings.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/pio-settings.md:6]: Exactly that, but it also prints a short description for each

File: pages/common/pio-settings.md (hunk 2 above)
Comment (marchersimon) [reply pages/common/pio-settings.md:14]: It doesn't create a new setting, just prints an error

Comment (bl-ue) [reply pages/common/pio-settings.md:14]: 👍🏻 check again

File: pages/common/pio-settings.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/pio-settings.md:10]: 👍🏻 check again

File: pages/common/pio-settings.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/pio-settings.md]: I just edited the page while you edited your suggestions. Not exactly the same, but hopefully close enough.

**Comment by marchersimon** [comment]
image
I think Austrian daylight saving time is a bit too much for Github :laughing:

**Comment by bl-ue** [comment]
Wow, that's truly bizzare coming from GitHub—what time does it say the first commit was made?

File: pages/common/pio-settings.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display the names, values and descriptions of all PlatformIO settings:
+
+`pio settings get`
+
+- Display the name, value and description of a specific PlatformIO setting:
+
+`pio settings get {{setting}}`
+
+- Modify the value of a setting:
```
Comment (navarroaxel) [inline pages/common/pio-settings.md:14]: Maybe this is better.
```suggestion
- Set a specific setting value:
```
or the word `configuration value `.

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages/common/pio-settings.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display the name, value and description of a specific PlatformIO setting:
+
+`pio settings get {{setting}}`
+
+- Set a specific setting value:
+
+`pio settings set {{setting}} {{value}}`
+
+- Reset the values of all modifed settings to their factory defaults:
```
Comment (bl-ue) [inline pages/common/pio-settings.md:18]: Oops, typo here: `modifed`

@marchersimon could we have a PR to fix this?

**Pull request merged**